src/test/java/ui/LoginTest.java

```diff
@@ -31,7 +31,7 @@ public void testNegativeLogin() {
         this.loginPage.verify();
         this.loginPage.login(email, password);
         String errorMessage = this.loginPage.getErrorMessage();
-        Assert.assertEquals(errorMessage, this.loginPage.getExpectedLoginErrMsg(), "The error message is " +
-            "not correct.");
+        Assert.assertEquals(errorMessage, this.loginPage.getErrorMessage(), "The error message is " +
```

Comment:
this.loginPage.getErrorMessage() trebuie sa fie this.loginPage.getExpectedLoginErrMsg()